core/templates/dev/head/components/sharing_links_directive.html

```diff
@@ -1,5 +1,30 @@
 <script type="text/ng-template" id="components/sharingLinks">
-  <ul class="oppia-sharing-links" layout="<[layoutType]>" layout-align="<[layoutAlignType]>">
+  <ul class="oppia-sharing-links" layout="<[layoutType]>" layout-align="<[layoutAlignType]>" ng-if="explorationsList.length > 0" ng-repeat="exploration in explorationsList">
```

Comment:
There is way too much duplication here, which is a big no-no, but I suspect this will not be necessary once you've gotten rid of explorationsList.